resource/temporary-resource-storage/build.gradle

```diff
@@ -31,6 +32,8 @@ dependencies {
 
     testImplementation project(':test-utils-common')
 
+    api "org.camunda.bpm:camunda-engine:${camundaVersion}"
```

Comment:
I think we generally want to restrict the camunda stuff to the core module. Do we actually need this?